mmv1/products/compute/RegionSecurityPolicy.yaml

```diff
@@ -193,3 +197,495 @@ properties:
+            Rules are evaluated from highest to lowest priority where 0 is the highest priority and 2147483647 is the lowest priority.
+          min_version: 'beta'
+          required: true
+          immutable: true
+        - name: 'match'
+          type: NestedObject
+          description: |
+            A match condition that incoming traffic is evaluated against.
+            If it evaluates to true, the corresponding 'action' is enforced.
+          min_version: 'beta'
+          properties:
+            - name: 'versionedExpr'
+              type: Enum
+              description: |
+                Preconfigured versioned expression. If this field is specified, config must also be specified.
+                Available preconfigured expressions along with their requirements are: SRC_IPS_V1 - must specify the corresponding srcIpRange field in config.
+              min_version: 'beta'
+              enum_values:
+                - 'SRC_IPS_V1'
+            - name: 'expr'
+              type: NestedObject
+              description: |
+                User defined CEVAL expression. A CEVAL expression is used to specify match criteria such as origin.ip, source.region_code and contents in the request header.
+              min_version: 'beta'
+              properties:
+                - name: 'expression'
+                  type: String
+                  description: |
+                    Textual representation of an expression in Common Expression Language syntax. The application context of the containing message determines which well-known feature set of CEL is supported.
+                  min_version: 'beta'
+                  required: true
+            - name: 'config'
+              type: NestedObject
+              description: |
+                The configuration options available when specifying versionedExpr.
+                This field must be specified if versionedExpr is specified and cannot be specified if versionedExpr is not specified.
+              min_version: 'beta'
+              properties:
+                - name: 'srcIpRanges'
+                  type: Array
+                  description: |
+                    CIDR IP address range. Maximum number of srcIpRanges allowed is 10.
+                  min_version: 'beta'
+                  item_type:
+                    type: String
+        - name: 'preconfiguredWafConfig'
+          type: NestedObject
+          description: |
+            Preconfigured WAF configuration to be applied for the rule.
+            If the rule does not evaluate preconfigured WAF rules, i.e., if evaluatePreconfiguredWaf() is not used, this field will have no effect.
+          min_version: 'beta'
+          properties:
+            - name: 'exclusion'
+              type: Array
+              description: |
+                An exclusion to apply during preconfigured WAF evaluation.
+              api_name: exclusions
```

Comment:
Just leaving a note here that these name/api_name diffs are consistent with SecurityPolicy's rule handling code, which I think it makes sense to match so that copy/pasting between the two is easier.